resources/js/components/Common/Button.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+
+const props = defineProps<{
+  small?: boolean
+  disabled?: boolean
+  text?: boolean
+  class?: string
+  type?: 'button' | 'submit' | 'reset' | undefined
+}>()
+
+const emit = defineEmits(['click'])
+</script>
+
+<template>
+  <button
+    v-if="!disabled"
+    class="font-semibold transition-colors duration-200 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary-600"
+    :class="`${small ? 'rounded-md text-xs 2xs:text-sm py-2' : 'rounded-lg py-3'} ${text ? 'text-black hover:text-primary-800 px-0 py-0' : 'px-2 2xs:px-3 bg-primary text-white hover:bg-primary-600 shadow-sm'} ${props.class}`"
```

Comment:
Same here, change class, and refactor ... `:class=[small ? ...`

Reply:
fixed